### PR TITLE
feat: rule to enforece 2 digit dates in e2e tests

### PR DESCRIPTION
**Pull request by harshal015**
## Description
app.clickup.com

Brief description of the changes made in this PR.

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Performance improvement
- [ ] Refactoring (no functional changes)

## Testing

- [ ] Tests added/updated for new functionality
- [ ] All existing tests pass
- [ ] Manual testing completed
- [ ] Edge cases tested

## Checklist

- [ ] Code follows the project's style guidelines
- [ ] Self-review of code completed
- [ ] Documentation updated (if applicable)
- [ ] No breaking changes (or breaking changes are documented)
- [ ] Changes are backward compatible
- [ ] Linting passes (`npm run lint`)
- [ ] Tests pass (`npm test`)
- [ ] Type checking passes (`npm run type-check`)

## Related Issues

Closes #(issue number)

## Screenshots (if applicable)

Add screenshots to help explain your changes.

## Additional Notes

Any additional information that reviewers should know.

### CODE REVIEW

File: packages/docs/rules/require-date-day-two-digits.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+new Date().toLocaleDateString('en-US', { month: 'short', day: '2-digit', year: 'numeric' });
+```
+
+### ❌ Invalid
+
+```js
+new Date().toLocaleDateString('en-US', { month: 'short', day: 'numeric', year: 'numeric' });
+```
+
+## Rationale
```
Comment (rvab) [inline packages/docs/rules/require-date-day-two-digits.md:25]: Configuration is missing

**Review by rvab — APPROVED** [review]

**Review by OmkarJ13 — APPROVED** [review]
